pkg/volume/local/local_test.go

```diff
@@ -76,6 +76,9 @@ func getPersistentPlugin(t *testing.T) (string, volume.PersistentVolumePlugin) {
 }
 
 func getTestVolume(readOnly bool, path string) *volume.Spec {
+	if err := os.MkdirAll(path, 0777); err != nil {
+		return nil
```

Comment:
would it work:

glog.Errorf("Failed creating folder for a test volume at %s (%v)", path, err)
return nil

I cannot return err, as this function return only volume Spec.